stdlib/scripts/check_license.mojo

```diff
@@ -0,0 +1,48 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+
+import sys
+from pathlib import Path
+
+# We can't check much more than this at the moment, because the license year
+# changes and the language is not mature enough to do regex yet.
+alias LICENSE = String(
+    """
+# ===----------------------------------------------------------------------=== #
+# Copyright (c)
+"""
+).strip()
+
+
+def main():
+    target_paths = sys.argv()
+    if len(target_paths) < 2:
+        raise Error("A file path must be given as a command line argument.")
+
+    one_file_failed = False
+    for i in range(len(target_paths)):
```

Comment:
**Thought** Wonder if it makes sense to add a `range` overload for `Sized` things like we do for `Intable` to avoid `range(len(x))` patterns.  Nothing to do here for this PR.  In this case, `sys.argv()` returns a `VariadicList` which implements `Sized`.

Reply:
in the future, we'll have iterators and replace all this by `for x in target_paths` so I don't think it's worth the effort